### DISCUSSION TITLE
Acessos para nova organização da Python Nordeste 2025 em Teresina-PI

### DISCUSSION THREAD

**Discussion by anapaulamendes** [apyb/comunidade#144] [Geral]
Oi, pessoal! Eu e @Emilly-horta somos as Big Kahunas da Python Nordeste 2025 que irá acontecer em Teresina, Piauí.

Estamos com um problema para acessar o e-mail mesmo tendo já a conta compartilhada do 1Password. Ao acessar o e-mail é solicitado um código no celular de alguém que o habilitou na autenticação de 2 fatores. Precisamos que seja removido para que a nova organização de Teresina consiga acessar sem dificuldade.

Precisamos ser adicionadas na organização do Github da Python Nordeste também para conseguir criar os repositórios e gerenciar os acessos da organização atual.

E no Telegram precisamos da permissão de administração.

Nos demais acessos também que não estejam no 1Password.

**Comment by itepifanio** [comment]
@anapaulamendes tentei remover o código no celular na conta google. Verifica se você já consegue acessar sem essa etapa adicional por favor

**Reply by itepifanio** [reply]
@anapaulamendes acabei de adicionar você e a Emilly como admin no grupo do telegram

**Reply by itepifanio** [reply]
@anapaulamendes conseguiu verificar o acesso a conta google, tem mais algum acesso faltante?

**Reply by anapaulamendes** [reply]
@itepifanio ainda está pedindo a autenticação pelo celular. Mas consegui acessar pelo passo que você sugeriu de ir em outro método de verificação e pedir o código no 1Password.

**Reply by itepifanio** [reply]
@anapaulamendes você conseguiria ver se consegue remover a autenticação pelo celular? Como você tem acesso ao google deveria conseguir.
Eu alterei a opção pra 1password e achei que teria resolvido, não resolveu. Removi o dispositivo e também não resolveu. Não sei mais qual seria o próximo passo.

**Reply by anapaulamendes** [reply]
Acho que a pessoa dona do celular quem precisa remover, porque se eu tenho a mesma permissão que você provavelmente não vou conseguir também.

**Reply by cecivieira** [reply]
@anapaulamendes sou eu. quando tu acessar, me manda dm no telegram que te dou acesso.
Acho que deve ser bug da Google, pois quando Ítalo configurou a 2. etapa no 1Password, a conta deslogou do meu celular e para realizar novo login eu usei o código gerado pelo 1Password para autenticar a segunda etapa.

**Reply by anapaulamendes** [reply]
@cecivieira eu consegui acessar gerando o código do 1Password. Mas a etapa de verificação pelo celular de alguém é realmente necessária tendo o 1Password? Como a conta vai passando de organização a cada ano, seria bom talvez ser gerenciado somente pelo gerenciador de senhas sem um vínculo no celular de uma pessoa específica. O que acham?

**Reply by cecivieira** [reply]
@anapaulamendes não é necessária.
o código o 1Password deveria substituir. É isso que eu e íTalo estamos falando

**Comment by anapaulamendes** [comment]
@cecivieira @gtsalles @Ryllari conseguem adicionar eu e @Emilly-horta no Github com as mesmas permissões que vocês?

**Reply by cecivieira** [reply]
Permissão dada para você e Emilly

**Reply by anapaulamendes** [reply]
Obrigada! 💙
